### DISCUSSION TITLE
CorsixTH 0.66-beta1

### DISCUSSION THREAD

**Discussion by TheCycoONE** [CorsixTH/CorsixTH#2356] [Team Posts]
I've cut a v0.66-beta1 tag and started a draft release page https://github.com/CorsixTH/CorsixTH/releases/tag/untagged-4e75ba0f49256b47cf11

Please let me know if you're participating in pre-releases and what dates work for you. I'd like to publish no later than the 28th.

**Comment by rmnvgr** [comment]
The beta is now available for Flatpak. Any date for the final release works for me.

**Comment by TheCycoONE** [comment]
CorsixTH 0.66-beta2 is now available as a draft page. There are only lua changes so it doesn't need to be recompiled. I plan to publish it tomorrow
